net-vpn/protonvpn-cli/protonvpn-cli-2.2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A VPN command-line tool from protonvpn - python rewrite"
+HOMEPAGE="https://protonvpn.com"
+SRC_URI="https://github.com/ProtonVPN/${PN}-ng/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
If the upstream calls it `-ng`, why won't you?

Reply:
It's installed as protonvpn-cli.  The idea behind the `-ng` seems to differentiate it from an earlier implementation in bash.  I don't see a problem with leaving the "ng" off, especially since the bash version was never added to any repos or overlays.